builds.yml

```diff
@@ -107,6 +108,7 @@ buildTypes:
       - APPLE_FLASK_CLIENT_ID
       - GOOGLE_CLIENT_ID_REF: GOOGLE_FLASK_CLIENT_ID
       - APPLE_CLIENT_ID_REF: APPLE_FLASK_CLIENT_ID
+      - GATOR_PERMISSIONS_ENABLED: true
```

Comment:
Is the environment variable needed? Ideally we can just use code fences.

Reply:
Yeah, I wasn't aware of the ability to use fences when we initially introduced the feature!

We are aiming to get this into the flask release cut tomorrow. I'd be happy if we can accept both the env var and the fence, and open a second PR to replace all checks on the env var to use the fence instead.

WDYT?